src/core/remote.js

```diff
@@ -1852,8 +1856,15 @@ Remote.prototype.createPathFind = function(options, callback) {
   }
 
   if (callback) {
+    const updateTimeout = setTimeout(() => {
```

Comment:
Why does pathfinding need a special case? Is it possible to just pass the timeout error generated on the request level?